src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -0,0 +1,145 @@
+import { BorderedRadio, Flex, RadioGroup, Text } from "@artsy/palette"
+import { useEffect, useState } from "react"
+import { OfferInput } from "v2/Apps/Order/Components/OfferInput"
+import { compact } from "lodash"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { PriceOptions_artwork } from "v2/__generated__/PriceOptions_artwork.graphql"
+import { appendCurrencySymbol } from "../Utils/currencyUtils"
```

Comment:
@jo-rs can we try tackling this Gravity tech debt after the end of January? it's only increasing in Force/Eigen

Reply:
What is this referring to, @araujobarret ? 🤔

Reply:
Isn't this `appendCurrySymbol` the USD thing from the other ticket?

Reply:
Oh, yes, sorry. Defo something to tackle in MP, but I think it's already somewhere on the board. Right, @ashleyjelks ? 🤔